clang_build/cli.py

```diff
@@ -131,7 +127,7 @@ def _main():
                 _setup_logger(None)
         else:
             _setup_logger(_logging.DEBUG)
-
+        _LOGGER.info("clang-build %s", _clang_build.__version__)
```

Comment:
any reason for not using `f"clang-build {_clang_build.__version__}"`?
Also, why put this here instead of `main`?

Reply:
The lazy evaluation of the logging platform means that the string is actually only generated, if the log level actually requires it.

I wanted the logger setup before I start logging.

Reply:
ups, I misplaced this line, it actually is inside `_main`